src/app/services/tvdb.service.ts

```diff
@@ -0,0 +1,22 @@
+import { Injectable } from '@angular/core';
+import { Observable } from 'rxjs/Observable';
+import { TvShowData } from '../tvshows/store/tvshows.models';
+
+const TVSHOWS: TvShowData[] = [
+  {title: 'WalkingDead'},
```

Comment:
Nit picking, but you might want to keep consistent spacing after curly braces. Using the style `{ title: 'WalkingDead' }` vs `{title: 'WalkingDead'}`

Reply:
Ok.  I'll also see if this is easy to catch this in tslint.